types/react-swipeable-bottom-sheet/package.json

```diff
@@ -0,0 +1,16 @@
+{
+    "private": true,
+    "name": "@types/react-swipeable-bottom-sheet",
+    "version": "1.1.9999",
+    "projects": [
+        "https://github.com/manufont/react-swipeable-bottom-sheet"
+    ],
+    "dependencies": {
+        "@types/react": "*",
```

Comment:
This package peer deps on React v16; does this actually work?

Reply:
You're right, I tested it with React 18 and didn't notice the peer dependencie.
Changed the package.json to require React ^16 and change the type definition to work with that.